rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -720,6 +730,16 @@ def _create_model_data(
 
         return model_data
 
+    def _remove_label_attribute_feature_sizes(
```

Comment:
This function can be decorated with `@staticmethod` since it does not use `self` variable anywhere. Pycharm gives you a warning for such functions. Subsequently, please add a test for this function.

Once you make the function static, the test can just call this function as `DIETClassifier._remove_label_attribute_feature_sizes(...)`

Reply:
> Once you make the function static, the test can just call this function as DIETClassifier._remove_label_attribute_feature_sizes(...)





Do you mean the ``test_convert_training_examples`` test in ``test_model_data_utils.py`` or the test that will be added in ``test_diet_classifier.py`` in the third PR? 
I think it should go to the second one because we separated the "label removing" logic from ``featurize_training_examples``.

Reply:
I mean a test for `_remove_label_attribute_feature_sizes` function alone in `test_diet_classifier.py`.